cylc/flow/post_install/__init__.py

```diff
@@ -0,0 +1,31 @@
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) NIWA & British Crown (Met Office) & Contributors.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+"""Plugins for running Python code after installation of a workflow.
+
+Built In Plugins
+----------------
+
+Cylc Flow provides the following post-install plugins:
+
+.. autosummary::
+   :toctree: built-in
+   :template: main_loop_plugin.rst
```

Comment:
This sets the template for autosummary to follow when building the pages, you may want to fall-back to the default template as the main loop template automatically documents all members.